pkg/builder/builder.go

```diff
@@ -620,7 +650,29 @@ func makeServerContainer(vdb *vapi.VerticaDB, sc *vapi.Subcluster) corev1.Contai
 		StartupProbe:    makeStartupProbe(vdb),
 		SecurityContext: makeServerSecurityContext(vdb),
 		Env:             envVars,
-		VolumeMounts:    buildVolumeMounts(vdb),
+		VolumeMounts:    buildServerVolumeMounts(vdb),
+	}
+	if vdb.IsSideCarDeploymentEnabled() {
+		cnt.Command = append(cnt.Command, buildVerticaStartCommand()...)
+	}
+	return cnt
+}
+
+// makeNMAContainer builds the spec for the nma container
+func makeNMAContainer(vdb *vapi.VerticaDB, sc *vapi.Subcluster) corev1.Container {
+	envVars := buildNMAEnvVars(vdb)
+	envVars = append(envVars, corev1.EnvVar{
+		Name:  NMALogPath,
+		Value: StdOut,
+	})
+
+	return corev1.Container{
+		Image:           pickImage(vdb, sc),
+		ImagePullPolicy: vdb.Spec.ImagePullPolicy,
+		Name:            names.NMAContainer,
```

Comment:
What about the resource for this container? How to we want to set it? I did not want to just set the same resource as the vertica container because I think they have different needs.